docker/Dockerfile

```diff
@@ -3,18 +3,22 @@ MAINTAINER Serenata de Amor "datasciencebr@gmail.com"
 
 USER root
 
-RUN apt-get update && apt-get install -y \
-  unzip
+RUN apt-get update \
+    && apt-get install unzip -y --no-install-recommends \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
 
 USER jovyan
 
 COPY requirements.txt ./
-COPY conda_requirements.txt ./
+RUN pip install --upgrade pip \
+    && pip install -r requirements.txt
 
-RUN pip install --upgrade pip
-RUN pip install -r requirements.txt
+COPY conda_requirements.txt ./
+RUN conda update --yes conda \
+    && conda config --add channels Rufone \
+    && conda config --add channels conda-forge \
+    && conda install --yes --file conda_requirements.txt \
+    && conda clean -tipsy
```

Comment:
Sorry, not sure what you meant =/

Just to make it clear, the idea with this change is to [remove the cache](http://conda.pydata.org/docs/commands/conda-clean.html) from the docker image and reduce the image size. The reason I said I dunno if it has any side effect is that the help says it should remove unused packages but I'm not sure how it detects unused packages since this all new to me 😄